drivers/SCsub

```diff
@@ -23,10 +23,8 @@ SConscript("coremidi/SCsub")
 SConscript("winmidi/SCsub")
 
 # Graphics drivers
-if env["platform"] != "server" and env["platform"] != "javascript":
+if env["vulkan"]:
     SConscript("vulkan/SCsub")
-else:
-    SConscript("dummy/SCsub")
```

Comment:
As discussed on chat, since this `SCsub` is removed it doesn't really make sense anymore to keep a `drivers/dummy` folder. Instead, we should move the respective dummy/null implementations close to their virtual class definitions (as was done already with `servers/audio/audio_driver_dummy.h`).

This can be done in a follow-up PR though.